portfolio/src/main/java/com/google/sps/servlets/CommentImageUrlServlet.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+
+import com.google.gson.Gson;
+import java.util.Map;
+import java.util.List;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet to get blobstore url for comment image upload **/
+@WebServlet("/commentsImageUrl")
```

Comment:
One of the good API design is not to use uppercase.
https://google.aip.dev/122#guidance

Maybe you can use `/comment-image-upload-url`?

Reply:
Just to clarify, according to https://google.aip.dev/122#collection-identifiers, collection identifiers must be in camelCase. Does this not apply for this case?

Reply:
Ah interesting. Lots of REST API guidance actually advocates hyphen-case.
This is new to me that they promote camel case.
I guess you don't need to do anything then :)

Reply:
Hmm but looking through the rest of the page they actually use hyphen-case so getting some mixed messages there? Actually on second thoughts looks like for individual components, hyphen-case is recommended, and for collections of components, camelCase is used, so basically it should be /camelCase/component-name.

Will change to hyphen case in a bit

Reply:
Collection identifiers use camelCase.
Resource IDs use hyphen-case.
In your case, it is a collection identifier.

Reply:
Actually many REST guidelines advocate hyphen-case.
So I guess just ensure that you're clear which guide that you follow and adhere strictly.